xinference/core/supervisor.py

```diff
@@ -102,46 +106,91 @@ def get_status(self) -> Dict:
             "workers": self._worker_status,
         }
 
+    @staticmethod
+    def _to_llm_reg(llm_family: "LLMFamilyV1", is_builtin: bool) -> Dict[str, Any]:
+        from ..model.llm import get_cache_status
+
+        specs = []
+        for spec in llm_family.model_specs:
+            cache_status = get_cache_status(llm_family, spec)
+            specs.append({**spec.dict(), "cache_status": cache_status})
+
+        return {**llm_family.dict(), "is_builtin": is_builtin, "model_specs": specs}
+
+    @staticmethod
+    def _to_embedding_model_reg(
```

Comment:
Ditto.

Reply:
Using an LRU cache might not be suitable for this scenario because the return value is subject to change.